index.js

```diff
@@ -6,14 +6,14 @@ const MIN_TIMEOUT = 10;
 const MAX_RETRIES = 5;
 const FACTOR = 6;
 
-module.exports = setup;
+module.exports = exports = setup;
 
 function setup(fetch) {
   if (!fetch) {
     fetch = require('node-fetch');
   }
 
-  function fetchRetry(url, opts = {}) {
+  async function fetchRetry(url, opts = {}) {
```

Comment:
```suggestion
  async function fetchRetry(url, opts = {}) {
```
(consequence of https://github.com/zeit/fetch-retry/pull/30/files#r386293072)